testing/features/oisc_warnings.feature

```diff
@@ -0,0 +1,46 @@
+Feature: OISC Warning components
+
+  The OISC Warning components allow content managers to provide options
+  to warn users about something important; in a variety of ways
+
+  Rule: OISC With Title and Description
+    Background:
+      Given a Title and Description OISC component is on the page
+
+    Scenario: Validate initial state
+      Then the OISC title is visible
+      And the OISC message is visible
+      And the OISC message contains a link
```

Comment:
I'd maybe not make the link part of the spec. As in usage links are totally optional. Not a big deal as this is a static state, but just reflecting that the link isn't _required_.

Reply:
Ah oki. No probs, I'll remove this. I wasn't 100% sure. So that main bit is just plain content then?

Reply:
Yep, limited formatting available in contentful, but consider that a bit of a black box of content.